module/Entra/Microsoft.Entra/DirectoryManagement/Get-EntraPartnerInformation.ps1

```diff
@@ -17,27 +17,14 @@ function Get-EntraPartnerInformation {
             $params["TenantID"] = $PSBoundParameters["TenantId"]
         }            
     
-            Write-Debug("============================ TRANSFORMATIONS ============================")
-            $params.Keys | ForEach-Object { "$_ : $($params[$_])" } | Write-Debug
-            Write-Debug("=========================================================================`n")
-            if ([string]::IsNullOrWhiteSpace($TenantId)) {
-                $TenantID = ((Invoke-MgGraphRequest -Method GET -Uri "/v1.0/organization").value).id
-            }
-            $response = Invoke-MgGraphRequest -Headers $customHeaders -Method GET -Uri "/v1.0/organization/$TenantID/partnerInformation"
-            # Create a custom table
-            $customTable = [PSCustomObject]@{
-                "PartnerCompanyName"       = $response.companyName
-                "companyType"              = $response.companyType
-                "PartnerSupportTelephones" = $response.supportTelephones
-                "PartnerSupportEmails"     = $response.supportEmails
-                "PartnerHelpUrl"           = $response.helpUrl
-                "PartnerCommerceUrl"       = $response.commerceUrl
-                "PartnerSupportUrl"        = $response.supportUrl
-                "ObjectID"                 = $response.partnerTenantId
-            }
-            $customTable 
+        Write-Debug("============================ TRANSFORMATIONS ============================")
+        $params.Keys | ForEach-Object { "$_ : $($params[$_])" } | Write-Debug
+        Write-Debug("=========================================================================`n")
+        if ([string]::IsNullOrWhiteSpace($TenantId)) {
```

Comment:
Would it be simpler to mark the **TenantId** parameter as Mandatory and add the `[ValidateNotNullOrEmpty()]` check in the parameter definition? You could also use a ValidateScript to check for whitespace and throw an error.